src/math/p_div.c

```diff
@@ -29,3 +29,33 @@ void p_div_f32(const float *a, const float *b, float *c,
         *(c + i) = *(a + i) / *(b + i);
     }
 }
+
+/**
+ *
+ * Element wise vector division between input vector 'a' and scalar 'b'
+ *
+ * @param a     Pointer to input vector
+ *
+ * @param b     Pointer to input scalar
+ *
+ * @param c     Pointer to output vector
+ *
+ * @param n     Size of 'a' and 'c' vector.
+ *
+ * @param p     Number of processor to use (task parallelism)
+ *
+ * @param team  Team to work with 
+ *
+ * @return      None
+ *
+ */
+
+void p_div_s(const float *a, const float *b, float *c,
+               int n, int p, p_team_t team)
+{
+
+    int i;
+    for (i = 0; i < n; i++) {
+        *(c + i) = *(a + i) / *(b);
```

Comment:
That's true for consts but in this case my recommendation was something like:

float inverse_b = 1.0f / *(b);
int i;
for (i=0; i < n; i++){
  *(c + 1) = *(a + i) * inverse_b;
}

and your accuracy concern comes into play only if there is an internal FPU representation of the division that keeps more precision than the 32bit float.  I can imagine thats possible but I've not tested it on the parallella hardware.  Do you know if it's the case?